doc/python-usage.rst

```diff
@@ -139,12 +142,25 @@ buffer allows, but only because dots are folded into their host character.
 Backlight Control
 ^^^^^^^^^^^^^^^^^
 These displays typically require a backlight to illuminate the liquid crystal
-display: by default GPIO 18 (PWM_CLK0) is used as the backlight control pin.
-This can  be changed by specifying ``gpio_LIGHT=n`` when initializing the
-device. The backlight can be programmatically switched on and off by calling
+display.  If the display's backlight is connected to one of the single-board
+computer's gpio pins, you can activate the backlight by specifying
+``gpio_LIGHT=n`` where n = the pin number when initializing the
+device (default GPIO 18 (PWM_CLK0)).
+
+If the display uses an I2C backpack with a pin from the backpack connected to
+the display's backlight pin, you can activate the backlight by specifying
+``backpack_pin=n`` where n = the pin number on the backpack.
+
+The backlight can be programmatically switched on and off by calling
 ``device.backlight(True)`` or ``device.backlight(False)`` respectively.
 
+.. note::
+  If you are using an I2C backpack based device, the backlight will not change
+  until the next time you send a command or data to the device.
+
 Examples
 ^^^^^^^^
-After installing the library, head over to the `luma.examples <https://github.com/rm-hull/luma.examples>`_ 
-repository. Details of how to run the examples is shown in the example repo's README.
+After installing the library, head over to the
+`luma.examples <https://github.com/rm-hull/luma.examples>`_
+repository. Details of how to run the examples is shown in the example repo's
```

Comment:
example repo's README sounds complicated. Maybe just 'README' and link to it directly?

Reply:
Changed as suggested.